src/qibolab/instruments/qblox/sequencer.py

```diff
@@ -9,6 +9,7 @@
 """Sampling rate for qblox instruments in GSps."""
 
 
+#
 class WaveformsBuffer:
     """A class to represent a buffer that holds the unique waveforms used by a
     sequencer.
```

Comment:
The main goal of this class is to deduplicate pulses. It will be replaced by #716

Reply:
However, it is not only doing that, but also checking memory and baking. These features should be preserved, but they might not require a dedicated class (or not specific for Qblox).